Libraries/LibWeb/HTML/HTMLMetaElement.cpp

```diff
@@ -160,4 +136,30 @@ void HTMLMetaElement::inserted()
     }
 }
 
+void HTMLMetaElement::removed_from(Node* parent)
+{
+    Base::removed_from(parent);
+
+    if (name().has_value()) {
+        if (name()->equals_ignoring_ascii_case("theme-color"sv)) {
+            document().obtain_theme_color();
+        } else if (name()->equals_ignoring_ascii_case("color-scheme"sv)) {
+            document().obtain_supported_color_schemes();
+        }
+    }
```

Comment:
Feels like this repeated bit of code should be pulled out into its own method.

Reply:
Okay, done :+1: